packages/shared-ux/avatar/solution/src/solution_avatar.stories.tsx

```diff
@@ -47,18 +57,21 @@ IconTypeAvatar.argTypes = {
   iconType: {
     control: 'select',
     options: [
+      'logoCloud',
+      'logoElasticStack',
       'logoElastic',
+      'logoElasticsearch',
       'logoKibana',
       'logoObservability',
-      'logoMaps',
-      'logoCloud',
       'logoSecurity',
+      'logoSiteSearch',
+      'logoWorkplaceSearch',
```

Comment:
I also mostly care here that we can display non-logo types as well. I would change this to show supporting different types of icons including logos, like:

```
['logoKibana', 'logoEnterpriseSearch', 'managementApp', 'machineLearningApp']
```

Reply:
Done.